src/Qdrant.Client/QdrantClient.cs

```diff
@@ -422,7 +422,14 @@ public class QdrantClient : IDisposable
 		TimeSpan? timeout = null,
 		CancellationToken cancellationToken = default)
 	{
-		await DeleteCollectionAsync(collectionName, timeout, cancellationToken).ConfigureAwait(false);
+		try
+		{
+			await DeleteCollectionAsync(collectionName, timeout, cancellationToken).ConfigureAwait(false);
+		}
+		catch (QdrantException)
+		{
+			// swallow this exception as it indicates the collection did not exist
```

Comment:
This will also swallow various unrelated exceptions (e.g. network problems - though those will presumably cause the CreateCollectionAsync below to throw as well).

Definitely sounds like a good idea to return a bool from DeleteCollecitonAsync, if qdrant returns that info...

Reply:
I _think_ this is only going to swallow where the service returns `false` in the result

https://github.com/qdrant/qdrant-dotnet/blob/9f0ae2e10a466c4ae3c1ea0d5f69f53f2fe122c2/src/Qdrant.Client/QdrantClient.cs#L530-L547

which looks like it is only returned from the server when the collection name didn't exist in the  hashmap (didn't exist at point of attempting to remove).

I do however think it makes sense to change `DeleteAsync` to return `Task<bool>`. This would be a breaking change to the API signature and to behaviour (no need to throw a `QdrantException` any more). What do you think about making this kind of change in a patch version @generall?

Reply:
Will merge this as is for now

Reply:
@russcam my vote would be to keep things a bit "fluid" for now, and allow some breaking changes which makes sense, as it's an early version etc. As more people use it and report issues, the SDK will stabilize etc.